### DISCUSSION TITLE
cvonline.lt: popups

### DISCUSSION THREAD

**Discussion by tired-creating-those-usernames** [uBlockOrigin/uAssets#23401] [Q&A] [answered]
Howdy, 
I have one site with a parasite banner popping up, it pops up even with Brave popups blocked and other similar settings in Ublock enabled;
when I do the manual element blocking I get nothing better than this (see screenshots)
maybe there is a way to get rid of this parasite?

![jobanas baneris2 cleaned](https://github.com/uBlockOrigin/uAssets/assets/166988440/078c9e1c-a2d3-4f64-93a1-97f99cfbdc43)
![jobanas baneris1 cleaned](https://github.com/uBlockOrigin/uAssets/assets/166988440/daa70b8b-ec51-41fe-8159-95b7e39f9168)

site: `https://cvonline.lt/lt`

**Comment by D4niloMR** [comment, marked as answer]
Try:

```adb
cvonline.lt##+js(set-cookie, seekerPopupBannerShown, true)
```
